dotnet/src/Experimental/Process.Core/ProcessStepBuilder.cs

```diff
@@ -29,7 +29,7 @@ public abstract class ProcessStepBuilder
     /// </summary>
     /// <param name="eventId">The Id of the event of interest.</param>
     /// <returns>An instance of <see cref="ProcessStepEdgeBuilder"/>.</returns>
-    public virtual ProcessStepEdgeBuilder OnEvent(string eventId)
+    public ProcessStepEdgeBuilder OnEvent(string eventId)
```

Comment:
not for this PR, but wondering if for the ProcessBuilder should have a custom implementation of this so only "public/external process events" could be subscribed to when a subprocess is a step

Reply:
I've noticed the _internal_ vs _public_ distiction as part of working on teh _Map Step_.  thb - I'm not entirely clear on the all the aspects of this distinction.  Is it possible for the caller to subscribe to _internal_ events?  Also, I'm unclear if the build is able to make that distinction at this juncture.  Great question, i just really have more questions than anything else.